src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -990,9 +990,11 @@ public void SetCookies(Uri uri, string cookieHeader)
         }
     }
 
+    // PathList needs to be public in order to maintain binary serialization compatibility as the System shim
+    // needs to have access to type-forward it.
     [Serializable]
     [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    internal struct PathList
+    public struct PathList
```

Comment:
While it might not there is still some potential compat issues (serialization or not) and I'd be nervous changing from struct to class in the future. Given it is a class in netfx I would make it a class in core as well.